packages/stellar_client/lib/src/client.dart

```diff
@@ -0,0 +1,486 @@
+          assetCode: 'TFT',
+          issuer: "GBOVQKJYHXRR3DX6NOX2RRYFRCUMSADGDESTDNBDS6CDVLGVESRTAC47",
+        );
+        break;
+      default:
+        break;
+    }
+
+    currencies = currency.Currencies({'TFT': tft});
+  }
+
+  Future<AccountResponse?> loadAccountFromPublicKey(
+      {required String accountId}) async {
+    try {
+      final account = await sdk.accounts.account(accountId);
+      keyPair = account.keypair;
+      return account;
+    } catch (error) {
+      print("Error Loading Account $error");
+      return null;
+    }
+  }
+
+  Future<KeyPair> loadAccountFromSecretSeed(
+      {required String secretSeed}) async {
+    keyPair = KeyPair.fromSecretSeed(secretSeed);
+    return keyPair;
+  }
+
+  Future<KeyPair?> createThreefoldAccount({required String mnemonic}) async {
+    try {
+      if (mnemonic.isNotEmpty) {
+        Wallet wallet = await Wallet.from(mnemonic);
+        keyPair = await wallet.getKeyPair();
+      } else {
+        keyPair = KeyPair.random();
+      }
+
+      Network networkType;
+      if (network == NetworkType.TESTNET) {
+        networkType = Network.TESTNET;
+      } else if (network == NetworkType.PUBLIC) {
+        networkType = Network.PUBLIC;
+      } else {
+        throw Exception("Unsupported network type");
+      }
+
+      Transaction? transaction = await getActivationTransaction();
+      if (transaction != null) {
+        transaction.sign(keyPair, networkType);
+        await sdk.submitTransaction(transaction);
+        print("Account Activated Successfully.");
+      } else {
+        throw Exception("Failed to retrieve activation transaction.");
+      }
+
+      await sdk.accounts.account(keyPair.accountId);
+
+      return keyPair;
+    } catch (error) {
+      throw Exception("Failed to create account $error");
+    }
+  }
+
+  Future<KeyPair?> createAccountFromSecret({required String secretSeed}) async {
+    try {
+      KeyPair existingAccountPair = KeyPair.fromSecretSeed(secretSeed);
+      keyPair = KeyPair.random();
+      AccountResponse existingAccount =
+          await sdk.accounts.account(existingAccountPair.accountId);
+
+      Transaction transaction = TransactionBuilder(existingAccount)
+          .addOperation(
+              CreateAccountOperationBuilder(keyPair.accountId, "10").build())
+          .build();
+
+      if (network == NetworkType.TESTNET) {
+        transaction.sign(existingAccountPair, Network.TESTNET);
+      } else if (network == NetworkType.PUBLIC) {
+        transaction.sign(existingAccountPair, Network.PUBLIC);
+      }
+
+      await sdk.submitTransaction(transaction);
+      await sdk.accounts.account(keyPair.accountId);
+      return keyPair;
+    } catch (error) {
+      print("Failed to create account $error");
+    }
+    return null;
+  }
+
+  // If testnet account activated with FriendBot, the asset should be added manually.
+  Future<bool> activateTestNetAccount({required String accountId}) async {
+    try {
```

Comment:
please check there that the network selected is testnet and throw an error if it's public